portal/templates/index.njk

```diff
@@ -33,6 +33,17 @@
 
     <div class="govuk-width-container">
 
+      {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+      {% if backLink and backLink.href %}
+        {{ govukBackLink({
+          text: "Back",
+          href: backLink.href,
+          attributes: {
+              "data-cy": "back-link"
```

Comment:
Indenting is slightly off here. Looks like 2 spaces in this file, though I see this is not consistent across all .njk files.